charts/hocs-audit/values.yaml

```diff
@@ -23,3 +23,4 @@ hocs-generic-service:
 
 dcuCaseView:
   enabled: false
```

Comment:
Should this is true?

Reply:
We turn this off by default and then enable it for all CS environments in the underlying environment deployment folders (I cannot remember the new repo's off the top of my head).

Reply:
Oh yeah of course, sorry!